Makefile

```diff
@@ -1,7 +1,20 @@
+.EXPORT_ALL_VARIABLES:
+
 GOLANGCI_LINT_CONFIG = .golangci.yml
 IMAGE_REPOSITORY ?= app-sre
 QUAY_USER ?=
 QUAY_TOKEN ?=
+OPM_VERSION = v1.14.0
+
+BUNDLE_DEPLOY_DIR=deploy/bundle
+MANIFEST_DIR=$(BUNDLE_DEPLOY_DIR)/manifests
+CSV=$(MANIFEST_DIR)/deploymentvalidationoperator.clusterserviceversion.yaml
+OPERATOR_IMAGE=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME)
+OPEARTOR_IMAGE_TAG=v$(OPERATOR_VERSION)
+BUNDLE_IMAGE?=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME)-bundle
+CATALOG_IMAGE?=$(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$(IMAGE_NAME)-catalog
+
```

Comment:
Can you define these from `OPERATOR_IMAGE`?

Reply:
Yep.  Good catch.  Done